tests/helpers/Stringer.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Behat\Mink\Tests\helpers;
```

Comment:
We always use singular names for our namespaces, and we start them with a capital letter